src/dc_msg.rs

```diff
@@ -178,15 +179,15 @@ pub unsafe fn dc_get_msg_info(context: &Context, msg_id: u32) -> *mut libc::c_ch
     }
     free(p as *mut libc::c_void);
 
-    if (*msg).type_0 != DC_MSG_TEXT {
+    if (*msg).type_0 != Viewtype::TEXT {
         ret += "Type: ";
         match (*msg).type_0 {
-            DC_MSG_AUDIO => ret += "Audio",
-            DC_MSG_FILE => ret += "File",
-            DC_MSG_GIF => ret += "GIF",
-            DC_MSG_IMAGE => ret += "Image",
-            DC_MSG_VIDEO => ret += "Video",
-            DC_MSG_VOICE => ret += "Voice",
+            Viewtype::AUDIO => ret += "Audio",
```

Comment:
Maybe add a `Display` trait to `Viewtype` instead of this?

Reply:
I see little value in it: this is only place of `Viewtype -> &str` conversion. Either way, it does not belong this commit.